ee-feature-pack/common/src/main/resources/modules/system/layers/base/com/fasterxml/jackson/jr/jackson-jr-objects/module.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2022, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<module xmlns="urn:jboss:module:1.9" name="com.fasterxml.jackson.jr.jackson-jr-objects">
```

Comment:
For any new module being added a decision needs to be made as to if we consider it public or private API.

However (this is an open question, not a request to make a change yet) - as this module is only depended on by a single module flagged as being private API I wonder if for now the artefact should be contained within the only module that actually makes use of it?

This may be more of a historic thing as well, we do tend to default to adding new modules but where there is only ever a 1:1 relationship do we really need the proliferation of modules.

Reply:
I tend to agree: If this is added ONLY for consumption by one module, I'd just add it as an artifact there and not create Yet Another Module until it's needed. If the dependency goes away, with the set up in this PR, we're likely going to end up with an orphaned module that someone will eventually find and clean up. ;)

Reply:
@darranl Thanks for your feedback. Looking at other potential dependencies it's only jackson.jr itself aside from opentelemetry. Integrating it into "io.opentelemetry.trace" is a good solution. I'm waiting for Jason's feedback and provide a consolidated PR afterwards.